kotlin-reference-server/src/main/kotlin/com/example/plugins/Sep24.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.plugins
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import mu.KotlinLogging
+
+val log = KotlinLogging.logger {}
+
+fun Route.sep24() {
+  route("/sep24/interactive") {
+    get {
+      log.info("Called /sep24/interactive with parameters ${call.parameters}")
+
+      val operation =
+        call.parameters["operation"]
+          ?: return@get call.respondText(
+            "Missing operation parameter",
+            status = HttpStatusCode.BadRequest
+          )
+
+      when (operation.lowercase()) {
+        "deposit" -> call.respondText("The sep24 interactive DEPOSIT starts here.")
+        "withdraw" -> call.respondText("The sep24 interactive WITHDRAW starts here.")
```

Comment:
So this PR is just to create the kotlin reference server's structure right? We also need to create functionality for providing off-chain status updates back to the anchor platform.

Reply:
Yes, I indent to do it in different PR to reduce number of changes